readme.md

```diff
@@ -44,6 +44,8 @@ var favicons = require('favicons'),
         orientation: "any",                       // Default orientation: "any", "natural", "portrait" or "landscape". `string`
         scope: "/",                               // set of URLs that the browser considers within your app
         start_url: "/?homescreen=1",              // Start URL when launching the application from a device. `string`
+        preferRelatedApplications: false,         // Should the browser prompt the user to install the native companion app. `boolean`
+        relatedApplications: undefined,           // Information about the native companion apps. This will only be used if `preferRelatedApplications` is `true`. `Array<{ id: string, url: string, platform: string }>`
```

Comment:
Why do not `relatedApplications: boolean | Array<string>`?

Reply:
Because maybe you want to define `related_applications` but not have chrome recommend them over the PWA prompt.